assets/src/domain/eventEditor/ui/datetimes/datesList/views/Grid.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { CSSProperties } from 'react';
+
+import DateCard from '../cardView/DateCard';
+
+const listStyle: CSSProperties = {
+	display: 'flex',
+	flexFlow: 'row wrap',
+	justifyContent: 'space-between',
+	width: '100%',
+};
+
+const Grid = ({ entities: datetimes }) => {
+	return (
+		<div style={listStyle}>
+			{datetimes.map((date) => (
+				<DateCard id={date.id} key={date.id} />
+			))}
+		</div>
+	);
+};
+
+export default Grid;
```

Comment:
This file should goto `/ui/datetimes/datesList/cardView` and should be named as `CardView`.

Reply:
Done: https://github.com/eventespresso/event-espresso-core/pull/2431/commits/1486caa876c4270d66db262b7df765404fd21603